ext/oci8/oci8_statement.c

```diff
@@ -1083,6 +1083,12 @@ int php_oci_bind_post_exec(zval *data)
 				}
 				break;
 		}
+	} else if ((Z_TYPE_P(bind->zval) == IS_TRUE) || (Z_TYPE_P(bind->zval) == IS_FALSE)) {
+		/* This convetrsion is done on purpose (ext/oci8 uses LVAL as a temorary value) */
+		if (Z_LVAL_P(bind->zval) == 0)
+			ZVAL_BOOL(bind->zval, FALSE);
+		else if (Z_LVAL_P(bind->zval) == 1)
+			ZVAL_BOOL(bind->zval, TRUE);
```

Comment:
@dstogov  Please replace "bind->zval" with "zv" from Line 1086 to Line 1091.

Reply:
@tianfyang Fixed now. Sorry for hurry.